baseline/commit-writer.yml

```diff
@@ -0,0 +1,13 @@
+---
+- hosts: localhost
+  tasks:
+    - name: Get Date and Time
```

Comment:
Can you please change this name to some more explanatory name like `Get Current Date And Time`